frontend/prisma/schema.prisma

```diff
@@ -10,27 +10,8 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Post {
+model Result {
   id        BigInt   @id @default(autoincrement())
```

Comment:
Pouzivat BigInt je tady nestastny, prisma ma built-in CUID().

`id           String   @id @default(cuid())`

Samotna databaze doporucuje pouzivat jejich built-in pro UUID, ale ten Prisma by default nema (museli bysme si napsat custom), takze to neni imo nutne.

viz: https://www.cockroachlabs.com/docs/stable/performance-best-practices-overview.html#unique-id-best-practices

Reply:
Migrace přidána